compiler/next/tools/chpldoc/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@
 # limitations under the License.
 
 add_executable(chpldoc "chpldoc.cpp")
+set_property(TARGET chpldoc PROPERTY CXX_STANDARD 17)
```

Comment:
> Note this "requires" C++17 only as a convenience step in prototyping
to get the fileystem functionality.

I'm not following? Won't this line make it fail to build if the compiler is not C++ 17 with this line? Is the idea that this tool is not always built?

Reply:
> Won't this line make it fail to build if the compiler is not C++ 17 with this line?
Correct
> Is the idea that this tool is not always built?
Previously and currently it is not always built.

Reply:
Let's put a comment here saying that this is temporary and that it needs to be removed when we get further